src/kokkos_ext/ArborX_DetailsKokkosExtViewHelpers.hpp

```diff
@@ -84,28 +84,43 @@ void reallocWithoutInitializing(ExecutionSpace const &space, View &v,
 }
 
 template <class ExecutionSpace, class View>
-typename View::non_const_type clone(ExecutionSpace const &space, View &v)
+typename View::non_const_type clone(ExecutionSpace const &space, View const &v,
+                                    std::string const &label)
 {
   static_assert(Kokkos::is_execution_space<ExecutionSpace>::value);
   static_assert(Kokkos::is_view<View>::value);
   typename View::non_const_type w(
-      Kokkos::view_alloc(space, Kokkos::WithoutInitializing, v.label()),
+      Kokkos::view_alloc(space, Kokkos::WithoutInitializing, label),
       v.layout());
   Kokkos::deep_copy(space, w, v);
   return w;
 }
 
+template <class ExecutionSpace, class View>
+typename View::non_const_type clone(ExecutionSpace const &space, View const &v)
+{
+  return clone(space, v, v.label());
```

Comment:
Wonder if a better default would be 
```suggestion
  return clone(space, v, v.label() +  "_clone");
```

Reply:
Merging as is and we can revisit later.
Recent use being added are all explicitly passing the label argument.
Changing the default calls for revisiting all previous usage which I would rather do just now.